### PR TITLE
Adding SQLServer schema collection information

### PR DESCRIPTION
**Pull request by edengorevoy**
### What does this PR do? What is the motivation?

We are able to display schema information in the frontend now, so this adds instructions to our SQLServer documentation for collecting schemas.

### CODE REVIEW

File: layouts/shortcodes/dbm-sqlserver-agent-config-examples.en.md [hunk 1]
```diff
@@ -71,6 +71,42 @@ instances:
       history_row_limit: 10000
 ```
 
+### Collecting schemas
+To enable this feature, use the `schemas_collection` option. Schemas will be collected on databases for which the Agent has `CONNECT` access.
```
Comment (rtrieu) [inline layouts/shortcodes/dbm-sqlserver-agent-config-examples.en.md:75]:
```suggestion
To enable this feature, use the `schemas_collection` option. Schemas are collected on databases for which the Agent has `CONNECT` access.
```

**Review by rtrieu — CHANGES_REQUESTED** [review]
Minor feedback for style consistency.

File: layouts/shortcodes/dbm-sqlserver-agent-config-examples.en.md [hunk 2]
```diff
@@ -71,6 +71,42 @@ instances:
       history_row_limit: 10000
 ```
 
+### Collecting schemas
+To enable this feature, use the `schemas_collection` option. Schemas are collected on databases for which the Agent has `CONNECT` access.
```
Comment (caseycull) [inline layouts/shortcodes/dbm-sqlserver-agent-config-examples.en.md:75]: Can we add min required Agent version here?

**Review by rtrieu — APPROVED** [review]

**Review by caseycull — APPROVED** [review]

**Pull request merged**